### DISCUSSION TITLE
iNAV Target Request for iFlight F7 blitz mini

### DISCUSSION THREAD

**Discussion by Grumpyrse** [iNavFlight/inav#8292] [General]
Hi all,
I see that there is a target for the full size version of this board, I tried flashing using that target but the gyro did not function correctly.
The latest version of the mini uses the BM1270, is this the problem or is this not supported at all.
If not, will this be implemented in the future?
Many thanks

**Comment by erstec** [comment]
> Hi all, I see that there is a target for the full size version of this board, I tried flashing using that target but the gyro did not function correctly. The latest version of the mini uses the BM1270, is this the problem or is this not supported at all. If not, will this be implemented in the future? Many thanks

https://github.com/iNavFlight/inav/blob/master/docs/policies/NEW_HARDWARE_POLICY.md
